server/controllers/student/addStudent.js

```diff
@@ -0,0 +1,22 @@
+const addStudent = require('../../database/queries/addStudent');
+
+exports.addStudent = (req, res) => {
+  addStudent({ ...req.body })
+    .then((student) => {
+      if (!student.rowCount) {
+        res.send({
```

Comment:
@AbdallahAmmar96 Here also you could set status to indicate an error.

Reply:
Done